src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta

```diff
@@ -213,10 +211,9 @@
                 this.element = $(Mustache.render(wrapper, config));
                 this.rePosition();
 
-                this.element.click(function (element) {
-                    completions.select($(element.target));
 
-                    completions.hideImmediate();
+                this.element.on("mouseup.sirius-autocomplete", ".autocomplete-row-js", function () {
```

Comment:
`this.element.find('.autocomplete-row-js').on('mouseup.sirius-autocomplete', function () {`